test/test_jit.py

```diff
@@ -4062,6 +4062,18 @@ def forward(self, x, w):
         with self.assertRaisesRegex(RuntimeError, 'test_jit.py\", line {}'.format(lineno + 3)):
             loaded(torch.rand(3, 4), torch.rand(30, 40))
 
+    def test_ignorable_args(self):
+        def fn():
+            val = torch.tensor([[1, 2, 3, 4, 9, 9], [1, 2, 3, 4, 9, 9]])
+            return val[2::, :1:]
```

Comment:
Can we write IR directly and use "define" to make it into a function to avoid making this test case fragile?

Something like this:
https://github.com/pytorch/pytorch/blob/2a178d34cd2acd04c05050689a25c79e777c28c5/test/test_jit.py#L5060

Reply:
sounds good